flash_notebooks/image_classification.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "transsexual-sense",
+   "id": "serious-guard",
```

Comment:
can we maybe omit these as only the id changed?

Reply:
The names are randomly generated for each cell. I hate it, but not sure how to resolve this without over-engineering.